internal/trivy/scan_job.go

```diff
@@ -133,7 +131,7 @@ func (f *filesystemScanJobBuilder) newImageScanJob(spec stasv1alpha1.ContainerIm
 
 	job.Spec.Parallelism = pointer.Int32(1)
 	job.Spec.Completions = pointer.Int32(1)
-	job.Spec.ActiveDeadlineSeconds = pointer.Int64(int64(ScanJobTimeout.Seconds()))
+	job.Spec.ActiveDeadlineSeconds = pointer.Int64(int64(7200))
```

Comment:
Why do we increase this timeout? Do we expect the scan job to take more than one hour?

Reply:
We cannot allow it to be shorter than the scan job timeout.

Reply:
Makes sense. I was just curious why we changed this now.

Reply:
We could set it to 1 hour and somehow document that scan job timeout over 1 hour does not work. WDYT?

Reply:
I have made an update, leaving the activeDeadlineSeconds value and lowering the scan job timeout to 30 minutes. PTAL!

Reply:
Agree, 1 hour should be more than enough.